src/GridFS/BucketReadWriter.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace MongoDB\GridFS;
+
+class BucketReadWriter
```

Comment:
Let's fold this class and its methods into the Bucket class, since that's where the spec expects to find these methods.